src/components/MasterDetails.js

```diff
@@ -56,6 +56,11 @@ type MasterDetailsProps = {
    * in the Table component for more details.
    */
   onSort: null | (sortColumn: number) => void;
+  /**
```

Comment:
Cool... this is more convenient than having to do the wrapped component on the outside... Can you move it so it's right under the details property? Also, maybe tweak the comment to say something like "Additional properties to pass to the details component in addition to the `data` property. Optional."